components/FormCard.js

```diff
@@ -138,28 +159,31 @@ export default function FormCard({ onAddTrip }) {
               </div>
               <ul>
                 {friends.map((friend, index) => (
-                  <li key={index}>
+                  <StyledList key={index}>
```

Comment:
You should never use a simple `index` value as key prop as they must be unique. Better combine a key value, e.g. 
```suggestion
                  <StyledList key={`${friend}-${index}`}>
```